Warning about redefinition of STDC_HEADERS when integrating glog

I get the following warning:

```
[  0%] Building CXX object packager/third_party/glog/CMakeFiles/glogbase.dir/src/demangle.cc.o
In file included from /shaka-packager/packager/third_party/glog/src/demangle.h:73,
                 from /shaka-packager/packager/third_party/glog/src/demangle.cc:39:
/shaka-packager/build/packager/third_party/glog/config.h:184: warning: "STDC_HEADERS" redefined
 #define STDC_HEADERS
 
In file included from /usr/include/c++/8/cstdio:41,
                 from /shaka-packager/packager/third_party/glog/src/demangle.cc:37:
/usr/include/x86_64-linux-gnu/c++/8/bits/c++config.h:1377: note: this is the location of the previous definition
 #define STDC_HEADERS 1
 ```

It appears to be because STDC_HEADERS is defined without a value.  If I change that in config.h.cmake.in, the warning goes away.

I'd like to be able to build [my project](https://github.com/shaka-project/shaka-packager) with warnings as errors, so I'd like to make glog compile cleanly without warnings.

--- a/src/config.h.cmake.in
+++ b/src/config.h.cmake.in
@@ -180,9 +180,6 @@
 /* The size of `void *', as computed by sizeof. */
 #cmakedefine SIZEOF_VOID_P ${SIZEOF_VOID_P}
 
-/* Define to 1 if you have the ANSI C header files. */
-#cmakedefine STDC_HEADERS 1
-
 /* the namespace where STL code like vector<> is defined */
 #cmakedefine STL_NAMESPACE ${STL_NAMESPACE}
 

--- a/src/config.h.cmake.in
+++ b/src/config.h.cmake.in
@@ -181,7 +181,7 @@
 #cmakedefine SIZEOF_VOID_P ${SIZEOF_VOID_P}
 
 /* Define to 1 if you have the ANSI C header files. */
-#cmakedefine STDC_HEADERS
+#cmakedefine STDC_HEADERS 1
 
 /* the namespace where STL code like vector<> is defined */
 #cmakedefine STL_NAMESPACE ${STL_NAMESPACE}

